src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
 body {
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
   margin: 0;
+  display: flex;
+}
+
+.stopwatch {
+  --minute-duration: 60s;
```

Comment:
Custom properties should be defined at the root level for better maintainability. Consider moving `--minute-duration` to the `:root` selector.